airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/destination/async/AsyncStreamConsumer.kt

```diff
@@ -54,6 +54,7 @@ constructor(
     workerPool: ExecutorService = Executors.newFixedThreadPool(5),
     private val airbyteMessageDeserializer: AirbyteMessageDeserializer =
         AirbyteMessageDeserializer(),
+    private val isFileTransfer: Boolean = false,
```

Comment:
nit: rename to `flushOnEveryMessage` (to reflect functionality rather than usage)

Reply:
Done